main.go

```diff
@@ -99,20 +116,45 @@ func runPrometheusServer(metricsAddr string, tlsOptions common.SSPTLSOptions) er
 	return nil
 }
 
-func getWebhookServer(sspTLSOptions common.SSPTLSOptions) *webhook.Server {
-	// If TLSSecurityProfile is empty, we want to return nil so that the default
-	// webhook server configuration is used.
-	if sspTLSOptions.IsEmpty() {
-		return nil
+// vm-console-proxy function
```

Comment:
Can you please add a link to that function? e.g. so I can compare the changes here and there, if there any changes.